cegs_portal/templates/pages/about.html

```diff
@@ -15,7 +15,11 @@
 {% block content %}
 
 <div class="container about-container flex flex-col space-y-2">
-    <h1>Duke Center for Combinitorial Gene Regulation Data Portal</h1>
+    <h1>Duke Center for Combinatorial Gene Regulation Data Portal</h1>
+
+    <div class="flex justify-center">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/rkF-t7BNzDY?si=x30Jg5qv_Imc92a4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
Does it still work if you remove some of the "allow"s? I don't think is needs accelerometer, autoplay, or gyroscope

Reply:
Removed and yes, it still works